lib/src/main/kotlin/MDP.kt

```diff
@@ -1,9 +1,11 @@
 abstract class MDP<TState, TAction> {
-    abstract fun transition(state: TState, action: TAction) : IDistribution<TState>
+
+    abstract fun transition(state: TState, action: TAction) : TState
```

Comment:
Why did these change? We should support probabilistic outcomes. If the game isn't probabilistic, it should just return a distribution with one single element.

Reply:
As discussed last meeting, we should aim to reduce the burden of implementing additional classes on the game side to implement, as MCTS does not explicitly require the probability class at the moment.